skimage/measure/_regionprops.py

```diff
@@ -166,6 +167,23 @@ def extent(self):
         return self.moments[0, 0] / (rows * cols)
 
     @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        from ._find_contours import find_contours
+        from scipy.spatial.distance import pdist
```

Comment:
Hm, this is annoying. We should not replace our label function. But it is not entirely clear to me why we still get a circular import. I'll have to check out your branch to see myself.

Thanks for going through this!